### PR TITLE
feat: add support for regional endpoints in STT

### PR DESCRIPTION
**Pull request by frsechet**
Finishes the work in #1

### CODE REVIEW

**Comment by frsechet** [comment]
@idevelop should be ready to merge now :-) Can you directly publish a new version of the package to npm? Thanks in advance!

File: README.md [hunk 1]
```diff
@@ -20,7 +20,11 @@ import { GoogleSpeechRecognition } from 'google-cloud-speech-webaudio';
 
 const GOOGLE_API_KEY = '...';
 
-const speechRecognition = new GoogleSpeechRecognition(GOOGLE_API_KEY);
+// Optionally define a regional endpoint: https://cloud.google.com/speech-to-text/docs/endpoints
+// If this parameter is not defined, the US endpoint will be used by default.
+const REGIONAL_ENDPOINT = 'https://eu-speech.googleapis.com'
```
Comment (idevelop) [inline README.md]: I would name this const something EU specific, make a US one next to it just for completeness sake, and reference the US const in the `speechToText` function signature and below.

**Review by idevelop — APPROVED** [review]
Approving anyway if you don't feel like changing the variable name, it's not a big deal. Thanks for fixing this!

File: README.md (hunk 1 above)
Comment (frsechet) [reply README.md]: OK for the first 2 points.
About referencing the US const in the speechToText, I think we should leave the default value, which is neither `eu-speech` nor `us-speech`: it's just `speech.googleapis...`

Comment (frsechet) [reply README.md]: see https://github.com/idevelop/google-cloud-speech-webaudio/blob/95b5d7d00a7f5d87e768d1c8f8284115669a4cec/src/speech-recognition.js#L24

either you set it to whatever you want, or it remains the default value and passes it down to the actual `speechToText()` call

**Comment by idevelop** [comment]
@frsechet version 0.1.4 published. Thanks again!

https://www.npmjs.com/package/google-cloud-speech-webaudio

**Comment by frsechet** [comment]
Cool, thanks for your work as well!
For context, we're going to use it in https://csml.dev 😃

**Pull request merged**